code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -202,17 +191,20 @@
 	desc = "A flask of holy water...it's been sitting in the Necropolis a while though."
 	list_reagents = list("hell_water" = 100)
 
-/obj/item/reagent_containers/food/drinks/holyoil
+/obj/item/reagent_containers/food/drinks/bottle/holyoil
 	name = "flask of zelus oil"
 	desc = "A brass flask of Zelus oil, a viscous fluid with a scenting of brass - this flask may be sipped or thrown."
 	icon_state = "zelusflask"
-	list_reagents = list("holyoil" = 30) //Powerfull
+	list_reagents = list("holyoil" = 30)
 	volume = 30
 	foodtype = NONE
-	force = 12 //Same as a toolbox
+	force = 18
 	throwforce = 18
+	knockdown_duration = 18
+	broken_force = 12
```

Comment:
I didnt want them to be used as broken weapons. As they are meant to be thrown and then done

Reply:
Confusing, but I'll remove the broken bottle extra force if so.